homeassistant/components/climate/strings.json

```diff
@@ -25,5 +25,21 @@
       "dry": "Dry",
       "fan_only": "Fan only"
     }
+  },
+  "state_attributes": {
+    "_": {
```

Comment:
Maybe we don't need attribute definitions per state class?

Reply:
I think it safer if we do have those, to be honest 👍 
(consistent)